operators/pkg/controller/elasticsearch/keystore/config.go

```diff
@@ -119,32 +118,32 @@ func NewConfigFromFlags() (Config, error, string) {
 		pass := viper.GetString(esPasswordFlag)
 
 		if user == "" {
-			return Config{}, errors.New("user is empty"), "invalid user"
+			return Config{}, errors.New("invalid user")
```

Comment:
I think we are losing a little information here. Why is the user invalid?
I would tend to keep only `errors.New ("user is empty")`.

Reply:
Previously, the `reason` field of the log message would have said "invalid user" which is what I tried to preserve. But I agree that it would be better to explain why the user is invalid.